.drone.yml

```diff
@@ -6,5 +6,5 @@ pipeline:
   test:
     image: cjsaylor/go-alpine-sdk:1.10
     commands:
-      - go get -u golang.org/x/tools/cmd/cover
-      - go test -cover $(go list ./... | grep -v /vendor/)
\ No newline at end of file
+      - GO111MODULE=on go get -u golang.org/x/tools/cmd/cover
+      - GO111MODULE=on go test -cover $(go list ./... | grep -v /vendor/)
```

Comment:
@DonRenando Instead of doing the "beta" route of go 1.10, what do you think of changing the `image` above to the 1.12 version.

I published a similar alpine sdk go image here: https://hub.docker.com/layers/cjsaylor/go-alpine-sdk/1.12/images/sha256-3f6ab6bcf7c21cf9a5951370801abe8ac95646eebbdb3061e2d13b43a98102b8.

IE, you should just be able to change the image from `cjsaylor/go-alpine-sdk:1.10` to `cjsaylor/go-alpine-sdk:1.12` and it should work.

Reply:
That's a good idea ! it's a better practice